packages/react/src/patterns/blocks/ContentBlockSimple/README.md

```diff
@@ -0,0 +1,103 @@
+import 'yourapplication.scss';
+
+function App() {
+  return (
+    <ContentBlockSimple
+      heading={heading}
+      copy={copy}
+      items={items}
+      mediaData={mediaData}
+      mediaType={mediaType}
+      cta={cta}
+    />
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the simplelongform styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+## Props
+
+| Name        | Required | Data Type | Default Value | Description                                                              |
+| ----------- | -------- | --------- | ------------- | ------------------------------------------------------------------------ |
+| `heading`   | YES      | String    | n/a           | Title of the content block.                                              |
+| `copy`      | YES      | String    | n/a           | Introduction copy briefly describing the section.                        |
+| `items`     | YES      | Array     | n/a           | Array of content items for use as content body.                          |
+| `mediaData` | NO       | Array     | n/a           | Array of image sizes for use with content items.                         |
+| `mediaType` | NO       | String    | 'image'       | Type of media. Currently only supports `image`.                          |
+| `cta`       | NO       | Object    | n/a           | CTA used at the end of content body. `Text` and `Card` styles supported. |
+
+### items
+
+| Name      | Data Type | Description          |
+| --------- | --------- | -------------------- |
+| `heading` | String    | Item section heading |
+| `copy`    | String    | Item section copy    |
+
+### mediaData
+
+| Name           | Data Type | Description                                            |
+| -------------- | --------- | ------------------------------------------------------ |
+| `src`          | String    | Url of image.                                          |
+| `minWidth`     | String    | Minimum width of image. Supports multiple breakpoints. |
+| `alt`          | String    | Image alt text.                                        |
+| `defaultImage` | String    | Default image to use.                                  |
+
+### CTA
+
+| Name    | Data Type | Description                                                  |
+| ------- | --------- | ------------------------------------------------------------ |
+| `style` | String    | CTA style. Supports `text` and `card`.                       |
+| `type`  | String    | CTA link behavior. Supports `external`, `jump`, and `local`. |
+| `title` | String    | CTA title.                                                   |
+| `href`  | String    | CTA url.                                                     |
+| `copy`  | String    | CTA copy.                                                    |
```

Comment:
I wouldn't bother repeating this documentation here. `ContentBlock` should indicate how it's limited, and `CTA` should offer what props are.

Reply:
I think I see what I did here... The CTA for this pattern comes from `ContentBlock` yeah? The children have no CTA at all.